src/cljs/athens/views.cljs

```diff
@@ -62,22 +62,22 @@
 
 (defn main
   []
-  (let [loading    (rf/subscribe [:loading?])
-        modal      (rf/subscribe [:modal])]
+  (let [loading (rf/subscribe [:loading?])
+        modal (rf/subscribe [:modal])
+        {:keys [msg type]} @(rf/subscribe [:db/snack-msg])]
```

Comment:
What's happening here?
Why do we need it with keybinding?

Reply:
Not related to keybindings. You asked me in another PR to move the subscription from line 70 to the let bindings at the start of the component. This removes the need of the function on line 70, and it can be just hiccup